app/services/solidus_square/webhooks/handlers/base.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "square"
+module SolidusSquare
+  module Webhooks
+    module Handlers
+      class Base < ::SolidusSquare::Base
+        attr_reader :params, :client
+
+        def self.call(params)
+          new(params).call
+        end
+
+        def initialize(params)
+          @params = params
+          @client = ::Square::Client.new(
+            access_token: SolidusSquare.config.square_access_token,
+            environment: SolidusSquare.config.square_environment
+          )
+          super
```

Comment:
Does this code look duplication of [SolidusSquare::Gateway#initializer](https://github.com/nebulab/solidus_square/blob/main/app/models/solidus_square/gateway.rb#L11-L14). Why you didn't use it rather than duplicate it?

Reply:
https://github.com/nebulab/solidus_square/pull/36/commits/1678ba76753920c509d9a378ff6173d76eb90f66